contrib/devtools/prepare_release/prepare_release.py

```diff
@@ -0,0 +1,542 @@
+def update_deprecation_height():
+    if (interactive):
+        config[k_approx_release_height] = input("Enter the approximate release height (mainnet): ")
+        config[k_weeks_until_deprecation] = input("Enter the weeks until deprecation: ")
+
+    lines = find_lines_containing(os.path.join(config[k_repository_root], "src/deprecation.h"), "static const int APPROX_RELEASE_HEIGHT.*=.*;")
+    if (len(lines) > 0):
+        approx_release_height_line = read_file_line(os.path.join(config[k_repository_root], "src/deprecation.h"), lines[0])
+    else:
+        print(f"Unable to find APPROX_RELEASE_HEIGHT in src/deprecation.h file")
+        sys.exit()
+    lines = find_lines_containing(os.path.join(config[k_repository_root], "src/deprecation.h"), "static const int WEEKS_UNTIL_DEPRECATION.*=.*;")
+    if (len(lines) > 0):
+        weeks_until_deprecation_line = read_file_line(os.path.join(config[k_repository_root], "src/deprecation.h"), lines[0])
+    else:
+        print(f"Unable to find WEEKS_UNTIL_DEPRECATION in src/deprecation.h file")
+        sys.exit()
+
+    approx_release_height_line = re.sub(r"(\d+)", str(config[k_approx_release_height]), approx_release_height_line).rstrip()
+    replace_file_line(os.path.join(config[k_repository_root], "src/deprecation.h"), 7, approx_release_height_line)
+
+    weeks_until_deprecation_line = re.sub(r"(\d+)", str(config[k_weeks_until_deprecation]), weeks_until_deprecation_line).rstrip()
+    replace_file_line(os.path.join(config[k_repository_root], "src/deprecation.h"), 8, weeks_until_deprecation_line)
+
+    blocks_per_hour = 24
+    blocks_per_day = 24 * blocks_per_hour
+    blocks_per_week = 7 * blocks_per_day
+    future_deprecation_height = int(config[k_approx_release_height]) + blocks_per_week * int(config[k_weeks_until_deprecation])
+
+    # Get last block time
+    last_block = get_last_blocks("https://explorer.horizen.io/")["blocks"][0]
+    last_block_height = int(last_block["height"])
+    last_block_timestamp = last_block["time"]
+    last_block_date = datetime.datetime.fromtimestamp(last_block_timestamp)
+
+    # Estimate the block time over the last 100'000 blocks
+    reference_block = get_block_from_height("https://explorer.horizen.io/", last_block_height - 100000)
+    reference_block_timestamp = reference_block["time"]
+    block_time = (last_block_timestamp - reference_block_timestamp) / 100000
+
+    # Estimate the datetime of the future_deprecation_height
+    future_deprecation_date = last_block_date + datetime.timedelta(seconds=(future_deprecation_height - last_block_height) * block_time)
+
+    # Estimate the next deprecation date
+    date_string = future_deprecation_date.strftime('%Y-%m-%d')
+
+    git_commit(f"Set deprecation height {future_deprecation_height}/{date_string}")
+
+def build_zend():
+    result_clean = subprocess.run(["make", "distclean"], cwd=config[k_repository_root])
+    result_build = subprocess.run([f"./zcutil/build.sh", "-j8"], cwd=config[k_repository_root])
```

Comment:
I guess this is a leftover, since you are not using any variable in this string:
```suggestion
    result_build = subprocess.run(["./zcutil/build.sh", "-j8"], cwd=config[k_repository_root])
```

Reply:
Yes you are right, modified accordingly.